install/image-references

```diff
@@ -26,3 +26,15 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-azure-machine-controllers:v4.0.0
+  - name: baremetal-operator
```

Comment:
operator is a misleading name here

Reply:
Changing the name here to metal3-baremetalhost-controller.  Some of the variable names internally used in the code are still using baremetal-operator.

Reply:
https://github.com/metal3-io/baremetal-operator/issues/227# captures discussion around (not) changing the name of the baremetal-operator. Based on that please let me know if you would still like us to give the image a name other than "baremetal-operator".